end-user-application-machines/new-agent-vm/rhel-standard-agent.json

```diff
@@ -99,7 +99,7 @@
             "metadata": {
                 "description": "Auto-generated token to access _artifactsLocation"
             },
-            "defaultValue": ""
+            "defaultValue": "?public"
```

Comment:
is the '?' a typo?

Reply:
not a typo, I need to provide a default value or the credential wont get serialized.

If i jsut leave it as public, it gets appended to the url and it fails the very first deployment because it uses github.com and appends "public" to the url which cases 404.

If we append ?public or really any query string, that does not cause a 404